docs/source/hub.rst

```diff
@@ -8,12 +8,14 @@ Publishing models
 Pytorch Hub supports publishing pre-trained models(model definitions and pre-trained weights)
 to a github repository by adding a simple ``hubconf.py`` file;
 
-``hubconf.py`` can have multiple entrypoints. Each entrypoint is defined as a python function with
-the following signature.
+``hubconf.py`` can have multiple entrypoints. Each entrypoint is a pre-trained model you want to
```

Comment:
I dont think it's always necessarily a pre-trained model you want to publish.
Maybe do:
Each entrypoint is defined as a python function (example: a pre-trained model you want to publish)

Reply:
Sounds good! Btw what else than pre-trained models you have in mind as an example of entrypoint?